mono/metadata/mono-route.c

```diff
@@ -153,7 +154,7 @@ gateway_from_rtm(struct rt_msghdr *rtm)
 	return 0;
 }
 
+#endif
 #endif
 
 extern const char mono_route_empty_file_no_warning;
-extern const char mono_route_empty_file_no_warning = 0;
```

Comment:
My mistake. The correct fix is to remove "extern" on the second line.
Otherwise you are going to get link/lib warnings.
The reason for the two lines is ridiculous gcc warnings.
Otherwise just the last line would suffice.